demos/object_detection_demo_ssd_async/README.md

```diff
@@ -112,14 +112,15 @@ Options:
     -h                        Print a usage message.
     -i "<path>"               Required. Path to a video file (specify "cam" to work with camera).
     -m "<path>"               Required. Path to an .xml file with a trained model.
-      -l "<absolute_path>"    Required for CPU custom layers. Absolute path to a shared library with the kernel implementations.
+      -l "<absolute_path>"    Required for CPU custom layers. Absolute path to a shared library with the kernels implementations.
```

Comment:
This is a) grammatically incorrect and b) unrelated to this PR.

Reply:
Reverted

Reply:
Submitted https://github.com/opencv/open_model_zoo/pull/386